suite/tests/CMakeLists.txt

```diff
@@ -3742,6 +3742,46 @@ foreach (run ${run_list})
   endif (enabled)
 endforeach(run)
 
+if (NOT DEBUG AND UNIX AND X64 AND NOT CMAKE_COMPILER_IS_CLANG)
+  set_tests_properties(
+    code_api|tool.drcacheoff.simple
+    code_api|tool.drcachesim.simple
+    code_api|tool.drcachesim.simple-config-file
+    code_api|tool.drcachesim.TLB-simple
+    code_api|tool.drcachesim.missfile
+    code_api|tool.drcachesim.missfile-config-file
+    code_api|tool.drcachesim.phys
+    code_api|tool.drcachesim.filter-simple
+    code_api|tool.drcachesim.filter-no-i
+    code_api|tool.drcachesim.filter-no-d
+    code_api|tool.drcachesim.delay-simple
+    code_api|tool.drcachesim.warmup-valid
+    code_api|tool.drcachesim.warmup-zeros
+    code_api|tool.drcachesim.threads
+    code_api|tool.drcachesim.threads-with-config-file
+    code_api|tool.drcachesim.TLB-threads
+    code_api|tool.drcachesim.multiproc
+    code_api|tool.drcachesim.miss_analyzer
+    code_api|tool.drcacheoff.simple
+    code_api|tool.drcacheoff.multiproc
+    code_api|tool.drcacheoff.filter
+    code_api|tool.drcacheoff.basic_counts
+    code_api|tool.drcacheoff.opcode_mix
+    code_api|tool.drcacheoff.view
+    code_api|tool.drcacheoff.burst_static
+    code_api|tool.drcacheoff.burst_replace
+    code_api|tool.drcacheoff.burst_replaceall
+    code_api|tool.drcacheoff.burst_noreach
+    code_api|tool.drcacheoff.burst_maps
+    code_api|tool.drcacheoff.burst_threads
+    code_api|tool.drcacheoff.burst_malloc
+    code_api|tool.drcacheoff.burst_threadfilter
+    code_api|tool.drcacheoff.burst_threadL0filter
+    code_api|tool.drcacheoff.burst_client
+    code_api|tool.drcacheoff.legacy
```

Comment:
Hmm, the reason the OSX label is done separately like this is because it was supposed to be temporary: once everything worked it would go away and that would be easier with it in once place.  This, however, is meant to be permanent, so I wonder if adding the label to separate individual tests is better?  I'm not sure.  This seems ok if you prefer this way.

Reply:
My reasoning was that it is much easier to see and change the list in case we want to adjust.

Reply:
The downside is that this list can get stale and out of sync with a new test being added or a test being removed.  If this were done locally, esp via a test attribute in the macros `torunonly_drcacheoff`, etc. that would all be taken care of automatically.  In any case this is fine for now.